.circleci/common.sh

```diff
@@ -97,7 +95,7 @@ function install_deps_pytorch_xla() {
 function build_torch_xla() {
   XLA_DIR=$1
   pushd "$XLA_DIR"
-  CC=gcc-7 CXX=gcc-7 python setup.py install
+  CC=clang-9 CXX=clang++-9 python setup.py install
```

Comment:
Let's leave a comment for future reference here, saying that the base image declared in `./circleci/config.yml` should have clang-9 installed. This is a critical info because different images would have different c compilers.

Reply:
Same comment as above -- this was part of the `revert` so we're just going back to the previous state. 

This is a good point though. I'll be working again to upgrade to cuda11, and I'll keep this in mind.